Makefile

```diff
@@ -1,5 +1,5 @@
 SKIP_SQUASH?=0
-VERSIONS="1"
+VERSIONS="1 slave-base slave-maven slave-nodejs"
```

Comment:
ah .. mean to ask about this before and got distracted .... so does 1 mean "master" image (the "traditional" image we have built), and the others obviously the slaves, where we couldn't change "1" to "master" because backward compatability?  Should we also have a "master" version or a "2" ?